### PR TITLE
Update demo to use only single app namespace 

### PR DESCRIPTION
**Pull request by kusumachalasani**
Update demo to 
- Run app on one namespace.
- Move the refresh metadata code into ReadMe
- Remove 3 mins sleep before generating the recommendations.

### CODE REVIEW

**Comment by kusumachalasani** [comment]
Finally, able to run the demo without issues using `quay.io/khansaad/autotune_operator:ds-auth` and tested the same. Works as expected!

File: monitoring/local_monitoring/local_monitoring_demo.sh [hunk 1]
```diff
@@ -129,139 +138,36 @@ function kruize_local() {
...
-  	echo ""
+	tfb_recommendations=$(curl -X POST "http://${KRUIZE_URL}/generateRecommendations?experiment_name=monitor_tfb_benchmark")
+	echo $tfb_recommendations
+	tfb_db_recommendations=$(curl -X POST "http://${KRUIZE_URL}/generateRecommendations?experiment_name=monitor_tfb-db_benchmark")
+	echo $tfb_db_recommendations
+
+	if [ -z "$tfb_recommendations" ] || [ -z "${tfb_db_recommendations}" ]; then
+		echo ""
+		echo "######################################################"
+		echo "RECOMMENDATIONS MIGHT BE EMPTY IF THERE IS NO ENOUGH DATA AVAILABLE!"
```
Comment (dinogun) [inline monitoring/local_monitoring/local_monitoring_demo.sh:156]: Please be specific about the minimum amount of data required. In our case we need atleast 2 data points to generate a recommendation

Comment (kusumachalasani) [reply monitoring/local_monitoring/local_monitoring_demo.sh]: Done.

**Review by chandrams — APPROVED** [review]
LGTM

**Pull request merged**